cabal.project

```diff
@@ -2,6 +2,9 @@ packages: *.cabal
 
 allow-newer: all
 
+package linear-base
+  ghc-options: -Wall -Werror
+
```

Comment:
This should really be only on CI, I think.

Reply:
I agree. However setting `--ghc-options` on command line for some reason applies those options to dependencies too, and they don't play nice with `-Werror`. I think the `cabal` issue is here: https://github.com/haskell/cabal/issues/3883 (And coincidently today is its 4th year anniversary :tada: ).

The workaround is to create a `cabal.project.ci` file, but I don't want to duplicate that file.

Reply:
Ok, let's merge as is, then.